deploy/manifests/dev/us-east-2/kustomization.yaml

```diff
@@ -16,6 +16,11 @@ secretGenerator:
   behavior: replace
   files:
   - peerkey.encrypted
+secretGenerator:
```

Comment:
Since there is already a `secretGenerator` we do not need to repeat this line. `secretGenerator` takes a list